src/licenseGen/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1 as build
+FROM mcr.microsoft.com/dotnet/sdk as build
```

can we fix the version to :5.0 if we are using netcoreapp5.0 below?

```suggestion
FROM mcr.microsoft.com/dotnet/sdk as build:5.0
```